WcaOnRails/lib/country_city_validators/gb_city_validator.rb

```diff
@@ -92,6 +92,17 @@
   Wiltshire
   Worcestershire
   Yorkshire
+  Akrotiri\ and\ Dhekelia
+  Anguilla
+  Bermuda
+  British\ Virgin\ Islands
+  Cayman\ Islands
+  Falkland\ Islands
+  Gibraltar
+  Montserrat
+  Pitcairn\ Islands
+  Saint\ Helena,\ Ascension\ and\ Tristan\ da\ Cunha
```

Comment:
Oh yikes! Turns out our `CityCommaRegionValidator` cannot handle counties with commas in them. I've fixed that (and a related bug) here: https://github.com/thewca/worldcubeassociation.org/pull/4254.

Reply:
I wonder if we'll have issues with this one if it's ever used, as it seems too large.

Reply:
Like, it's so long our database will reject the field?

Reply:
I don't know where exactly the restriction is, but if you start writing something on the city field at some point it just won't let you write more.